recipes/zippy/all/conanfile.py

```diff
@@ -0,0 +1,31 @@
+from conans import ConanFile, tools
+import os
+
+class NoWideConan(ConanFile):
+    name = "zippy"
```

Comment:
It looks like there is more then one `zippy` project on github. Perhaps it's safer to namespace it?
This will also require a folder rename.
```suggestion
    name = "troldal-zippy"
```